bindings/nodejs/package.json

```diff
@@ -59,15 +59,12 @@
     "node": ">= 10"
   },
   "scripts": {
-    "artifacts": "napi artifacts",
-    "build": "napi build --platform --release --js generated.js && node ./scripts/header.js",
-    "build:debug": "napi build --platform --js generated.js && node ./scripts/header.js",
+    "build": "napi build --platform --target \"${NAPI_TARGET:-}\" --release --js generated.js && node ./scripts/header.js",
+    "build:debug": "napi build --platform --target \"${NAPI_TARGET:-}\" --js generated.js && node ./scripts/header.js",
     "docs": "typedoc",
     "format": "prettier --write .",
-    "prepublishOnly": "napi prepublish -t npm",
     "test": "cucumber-js",
-    "bench": "node ./benchmark/index.js",
-    "version": "napi version"
```

Comment:
This convenient command can automatically synchronize the version number of package.json to all sub-packages. We can keep it.

Reply:
No, no needed.